src/cmd/services/m3coordinator/ingest/carbon/ingest.go

```diff
@@ -127,18 +126,20 @@ func NewIngester(
 		}
 	})
 
-	return &ingester{
+	i := &ingester{
```

Comment:
meganit; `ingest` as name maybe? `i` is usually an index or something of that sort